bundles/org.openhab.core.io.transport.modbus/pom.xml

```diff
@@ -35,7 +35,7 @@
     <dependency>
       <groupId>net.wimpi</groupId>
       <artifactId>jamod</artifactId>
-      <version>1.2.4.OH</version>
+      <version>1.2.5.OH-SNAPSHOT</version>
```

Comment:
You want to delay this PR and release jamod first.

Reply:
> You want to delay this PR and release jamod first.

Hmm. It is actually a rather complicated process..

- the old core transport code will not build with the new jamod version jar
- the new core transport code will not build with the old jamod version jar
- ditto both ways for the new/old binding code with the new/old core transport code

The conclusions are..

1. The three PR's must be released in the same release cycle (say 3.0.1)
2. The build sequence is 1) build new jamod, 2) build new transport, and 3) build new binding.
3. Personally I don't know how to handle this in Jenkins validation checks, or publishing it in JFrog ;) ??
4. The POM file names must be synchronized all the way through (i.e. remove the 'SNAPSHOT' tags).

Reply:
When jamod pr is accepted, release can be made from jamod only.

Then this PR should refer to that new jamod version

Reply:
Correct. I am assuming the maintainers will read these comments. :)

Reply:
PS do you have any preference concerning the new version number? 1.2.5 / 1.3.0 / or even 3.0.0 ??

Reply:
No preference really. I guess you could consider it major feature since the API is different with the new parameter?

Reply:
Yes I was thinking of 1.3.0 too, as it is more than just a bug fix. I will modify the POMs..